src/workerd/io/compatibility-date.capnp

```diff
@@ -648,7 +648,7 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   pythonWorkers20250116 @71 :Bool
       $compatEnableFlag("python_workers_20250116")
-      $experimental
+      $impliedByAfterDate(name = "pythonWorkers", date = "2025-09-12")
```

Comment:
No reason to only do this after that date. 

```suggestion
      $impliedByAfterDate(name = "pythonWorkers", date = "2000-01-01")
```

Reply:
I think there is a reason though? We don't want this to be effective immediately?

Reply:
hmm, okay, I read this annotation the wrong way around.

Reply:
Alright, I had some concerns about our tests which check for presence of the `pythonWorkers20250116` flag. But they should be unaffected until we change their compat date.

Sorry for the noise.